jobs/issues_status.rb

```diff
@@ -10,8 +10,8 @@
   opened_series = [[], []]
   closed_series = [[], []]
   issues_by_period = backend.issue_count_by_status(
-    orgs: (ENV['ORGS'].split(',') if ENV['ORGS']),
-    repos: (ENV['REPOS'].split(',') if ENV['REPOS']),
+    orgs: (ENV['ORGS'].split(',')),
+    repos: (ENV.fetch('REPOS', '').split(',')),
```

Comment:
It's just a habit. In this case, both work equally, but in some cases, using `||` instead of `fetch` with a default value can bite you, as explained here: http://devblog.avdi.org/2009/03/16/go-fetch/